app/src/main/java/fr/free/nrw/commons/coordinates/CoordinateEditHelper.kt

```diff
@@ -23,7 +23,7 @@ import timber.log.Timber
 
 /**
  * Helper class for edit and update given coordinates and showing notification about new coordinates
- * upgradation
+ * gradation
```

Comment:
I guess the original comment meant `notification about new coordinates having been set`. Would you mind changing to that?